src/main/groovy/org/mockito/release/gradle/ReleaseConfiguration.java

```diff
@@ -367,7 +380,7 @@ private String getString(String key, String envVarName) {
     }
 
     private Map getMap(String key) {
-        return (Map) getValue(key, null,"Please configure 'releasing." + key + "' value (Map).");
+        return (Map) getValue(key, null, "Please configure 'releasing." + key + "' value (Map).");
```

Comment:
you can define a constant: `private static final String NO_ENV_VARIABLE = null` and pass it instead of `null`. Then it will be easier to read this line